<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 0, 83);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(17, 2, 47);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_error">
         <property name="maximumSize">
          <size>
           <width>275</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(36, 189, 100);
border-radius: 8px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_erro">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
color: rgb(17, 2, 47);
</string>
            </property>
            <property name="text">
             <string>Bem-vindo!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_erro">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QAbstractButton{
	color: rgb(17, 2, 47);
	border: 0;
	border-radius: 8px;
}
QAbstractButton::hover{
	color: white;
}</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="main">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(17, 2, 47);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>335</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(17, 2, 47);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="titulo">
          <property name="geometry">
           <rect>
            <x>67</x>
            <y>30</y>
            <width>200</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Berlin Sans FB Demi</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Express√£o</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_login">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>280</y>
            <width>270</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Berlin Sans FB Demi</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QAbstractButton{
	border: 0;
	border-radius: 10px;
	background-color: rgb(36, 189, 100);
	color: rgb(17, 2, 47);
}
QAbstractButton::hover{
	background-color: rgb(32, 171, 90);
}</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_email">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>110</y>
            <width>270</width>
            <height>60</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Berlin Sans FB</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(17, 2, 47);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_email">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>0</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB Demi</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_email">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>20</y>
             <width>260</width>
             <height>35</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	border: 0; 
	border-bottom: 1px solid rgb(139, 139, 139);
	color: rgb(139, 139, 139);
}
</string>
           </property>
           <property name="placeholderText">
            <string>insira seu email</string>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="frame_senha">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>170</y>
            <width>270</width>
            <height>60</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(17, 2, 47);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_senha">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>0</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB Demi</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Senha</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_senha">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>20</y>
             <width>260</width>
             <height>35</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	border: 0; 
	border-bottom: 1px solid rgb(139, 139, 139);
	color: rgb(139, 139, 139);
}</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>insira sua senha</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="bt_forgetpassword">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>240</y>
            <width>141</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Berlin Sans FB</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QAbstractButton{
	border: 0;
	color: rgb(139, 139, 139)
}
QAbstractButton:hover{
	text-decoration:underline;
}</string>
          </property>
          <property name="text">
           <string>Esqueceu sua senha?</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_cadastrar">
          <property name="geometry">
           <rect>
            <x>97</x>
            <y>350</y>
            <width>141</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Berlin Sans FB</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QAbstractButton{
	border: 0;
	color: rgb(36, 189, 100);
}
QAbstractButton::hover{
	text-decoration: underline;
}</string>
          </property>
          <property name="text">
           <string>Cadastre-se</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
